tethys_apps/templates/tethys_apps/app_base.html

```diff
@@ -103,6 +103,79 @@
       {% gizmo_dependencies global_js %}
     {% endblock %}
 
+    {% block analyticsscripts %}
+      {% if analytics.analytical is True %}
```

Comment:
@rileyhales I'm finally getting around to reviewing this. Sorry for the delay. Although not entirely wrong, the "is True" blocks seem suspect. Try changing all the statements that art {% if <var> is True %} to {% if <var> %} or {% if <var> == True %} if you only want to check for whether the value is True.